File: Stete-app/app/src/main/java/tzilic_20/stete/model/Steta.java
package tzilic_20.stete.model;

import java.io.Serializable;
import java.util.Date;

public class Steta implements Serializable {

    private int sifra;
    private String vlasnik;
    private float geografskaSirina;
    private float geografskaDuzina;
    private float iznosStete;

    public Steta() {
    }

    public Steta(int sifra, Date datum, String kolegij, int brojPrijavljenih) {
        this.sifra = sifra;
        this.vlasnik = vlasnik;
        this.geografskaSirina = geografskaSirina;
        this.geografskaDuzina = geografskaDuzina;
        this.iznosStete = iznosStete;
    }

    public int getSifra() {
        return sifra;
    }

    public void setSifra(int sifra) {
        this.sifra = sifra;
    }

    public String getVlasnik() {
        return vlasnik;
    }

    public void setVlasnik(String vlasnik) {
        this.vlasnik = vlasnik;
    }

    public float getGeografskaSirina() {
        return geografskaSirina;
    }

    public void setGeografskaSirina(float geografskaSirina) {
        this.geografskaSirina = geografskaSirina;
    }

    public float getGeografskaDuzina() {
        return geografskaDuzina;
    }

    public void setGeografskaDuzina(float geografskaDuzina) {
        this.geografskaDuzina = geografskaDuzina;
    }

    public float getIznosStete() {
        return iznosStete;
    }

    public void setIznosStete(float iznosStete) {
        this.iznosStete = iznosStete;
    }
}

File: potres-api/src/tzilic_20/Steta.php
<?php
namespace tzilic_20;

/**
 * @Entity @Table(name="prijava_stete")
 **/
class Steta
{
    /** @Id @Column(type="integer") @GeneratedValue  **/
    private $sifra;

    /** @Column(type="string") **/
    private $vlasnik;

    /** @Column(type="float") **/
	private $geografskaSirina;

	 /** @Column(type="float") **/
	 private $geografskaDuzina;

	  /** @Column(type="float") **/
	  private $iznosStete;
	
	public function __construct($podaci=null)
	{
		if($podaci==null){
			return;
		}
		$this->vlasnik=$podaci->vlasnik;
		$this->geografskaSirina=$podaci->geografskaSirina;
		$this->geografskaDuzina=$podaci->geografskaDuzina;
		$this->iznosStete=$podaci->iznosStete;
	}

	public function getSifra(){
		return $this->sifra;
	}

	public function setSifra($sifra){
		$this->sifra = $sifra;
	}

	public function getVlasnik(){
		return $this->vlasnik;
	}

	public function setVlasnik($vlasnik){
		$this->vlasnik = $vlasnik;
	}

	public function getGeografskaSirina(){
		return $this->geografskaSirina;
	}

	public function setGeografskaSirina($geografskaSirina){
		$this->geografskaSirina = $geografskaSirina;
	}

	public function getGeografskaDuzina(){
		return $this->geografskaDuzina;
	}

	public function setGeografskaDuzina($geografskaDuzina){
		$this->geografskaDuzina = $geografskaDuzina;
	}

	public function getIznosStete(){
		return $this->iznosStete;
	}

	public function setIznosStete($iznosStete){
		$this->iznosStete = $iznosStete;
	}

}